### ISSUE TITLE
Cannot remove package from nexus repository; creates a 404 error.

### ISSUE THREAD

**Issue by JigsawCorp** [sonatype-nexus-community/nexus-repository-conan#59]
* **What are you trying to do?**
I am trying to remove a recipe and its packages from a nexus repo.
* **What feature or behavior is this required for?**
This is required to remove a recipe and its packages from a nexus repo.
* **How could we solve this issue? (Not knowing is okay!)**
I have a feeling this is related to the search function not working on nexus repositories, but I have no clue what could cause this.

Whenever I try to remove a package from my nexus repo, I get a 404:

`conan remove PackageTest/1.0.0@myuser/testing -r=my_private_repo`

Output:
`ERROR: 404 Not found. [Remote: my_private_repo]`

* **Anything else?**

**Comment by memsharded** [comment]
What is the output of ``conan search PackageTest/1.0.0@myuser/testing -r=my_private_repo``? And of ``conan search PackageTest* -r=my_private_repo``? 

Can you install the package normally? You can try removing first PackageTest from your local cache, then issuing a ``conan install PackageTest/1.0.0@myuser/testing -r=my_private_repo``.

**Comment by JigsawCorp** [comment]
@memsharded 

Some information from the logs have been replaced for confidentiality reasons, but the format is the same. All changed info is like this [MY USERNAME] or [OUR URL].

The output of `conan search PackageTest/1.0.0@myuser/testing -r=my_private_repo` is 
{"_action": "REST_API_CALL", "duration": 0.018169164657592773, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.15.0 (Python 2.7.15rc1) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********", "X-Client-Id": "[MY USERNAME]"}, "method": "GET", "time": 1557411997.957008, "url": "https://[OUR URL]/repository/conan/v1/conans/PackageTest/1.0.0/myuser/testing/search"}
{"_action": "EXCEPTION", "class": "NotFoundException", "message": "[OUR 404 HTML CODE GOES HERE] [Remote: my_private_repo]", "time": 1557411997.957535}

That was tested on Python 2.7.15. I will try with >= 3.7 and report back.

The result of installing the package after having it removed is:
```
$ conan install PackageTest/1.0.0@myuser/testing -r=my_private_repo
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++
compiler.version=7
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

PackageTest/1.0.0@myuser/testing: Retrieving from server 'my_private_repo' 
PackageTest/1.0.0@myuser/testing: Trying with 'my_private_repo'...
Downloading conanmanifest.txt
[==================================================] 58B/58B          
Downloading conanfile.py
[==================================================] 422B/422B        
PackageTest/1.0.0@myuser/testing: Downloaded recipe revision 0
Installing package: PackageTest/1.0.0@myuser/testing
Requirements
    PackageTest/1.0.0@myuser/testingfrom 'facilis' - Downloaded
Packages
    PackageTest/1.0.0@myuser/testing:0ab9fcf606068d4347207cc29edd400ceccbc944 - Download

PackageTest/1.0.0@myuser/testing: Retrieving package 0ab9fcf606068d4347207cc29edd400ceccbc944 from remote 'my_private_repo' 
ERROR: Binary package not found: 'PackageTest/1.0.0@myuser/testing:0ab9fcf606068d4347207cc29edd400ceccbc944'
```

The log entry right before the exception is:
{"_action": "REST_API_CALL", "duration": 0.024103879928588867, "headers": {"Authorization": "**********", "User-Agent": "Conan/1.15.0 (Python 2.7.15rc1) python-requests/2.21.0", "X-Client-Anonymous-Id": "**********", "X-Client-Id": "[MY USERNAME]"}, "method": "GET", "time": 1557412327.249027, "url": "https://[OUR URL]/repository/conan/v1/conans/PackageTest/1.0.0/myuser/testing/packages/0ab9fcf606068d4347207cc29edd400ceccbc944"}

It seems like all http calls are to v1, and from my understanding v2 fixed some issues? If I am correct, how would I go about installing the v2? Is this something server-side or is this client-side?

Thank you for your time.

**Comment by memsharded** [comment]
Hi @JigsawCorp 

No, this should work correctly, even if using only V1. Conan still does support and works fine with V1, and it will surely for quite a long time, it is still the most used API out there, as V2 is released but opt-in at this moment. 

It seems it might be some failure in the implementation of V1 by nexus, but I am sorry that I am a bit lost there, we would need someone from @sonatype-nexus-community to have a look.

**Comment by JigsawCorp** [comment]
@memsharded  

Thanks anyways, do you think using an older version would help? Seems like someone said using 1.13.3 worked. 

The problem is that I tried doing a clean install of any older version of conan on Windows 10 and Ubuntu, and I kept getting this error:
`ERROR: Conan storage path has to be an absolute path`

Any idea what this could be?

**Comment by Liskan** [comment]
I think the delete (like the search) functionality was not implemented yet.

**Comment by memsharded** [comment]
@JigsawCorp There was a change in how the storage is represented in the ``conan.conf`` file. Try removing your ``<userhome>/.conan`` folder completely and restart, and tell if it is still failing.

**Comment by igorudovika** [comment]
Conan remove works fine in current released 1.0.2 version. I am closing this one. Please open another one if you still need help with remove command.
